share/spice/cricket/cricket.js

```diff
@@ -0,0 +1,159 @@
+        LIVE_SCORECARD_ENDPOINT = 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20cricket.scorecard.live&format=json&env=store%3A%2F%2F0TxIGQMQbObzvU4Apia0V0&callback=';
+    env.ddg_spice_cricket = function (api_result) {
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.query.results) {
+            return Spice.failed('cricket');
+        }
+        //TODO:select latest series
+        var data = api_result.query.results.Series.constructor === Array ? api_result.query.results.Series[0] : api_result.query.results.Series;
+        DDG.require("moment.js", function () {
+            // Render the response
+            Spice.add({
+                id: "cricket",
+                name: "Cricket",
+                data: data.Schedule.Match,
+                meta: {
+                    //TODO: update display
+                    primaryText: "<span style='color:deepskyblue;'>" + data.SeriesName + ", Schedule & Score</span>",
+                    idField: "matchid",
+                    selectedItem: getSelected(data.Schedule.Match),
+                    scrollToSelectedItem: true,
+                    itemsHighlight: true,
+                    itemsExpand: true,
+                    //secondaryText: '<span class="tx-clr--grey-dark">YQL</span>',
+                    rerender: ["teams.0", "teams.1"],//TODO: need a better way
+                    //sourceName: "developer.yahoo.com",
+                    //sourceUrl: 'http://example.com/url/to/details/' + api_result.name
+                },
+                normalize: function (item) {
+                    var winner = getWinner(item);
+                    return {
+                        matchid: item.matchid,
+                        mtype: item.mtype.toUpperCase(),
+                        matchNo: item.MatchNo,
+                        venue: item.Venue.content,
+                        date: moment.utc(item.StartDate).local().format('ddd, Do MMM'),
+                        relativeTime: moment.utc(item.StartDate).local().calendar(),
+                        startTime: moment.utc(item.StartDate).local().format('LT'),
+                        startDate: Date.parse(moment.utc(item.StartDate).local().format("YYYY-MM-DD")),
+                        matchTimeSpan: item.MatchTimeSpan,
+                        teams: item.Team,
+                        result: item.Result,
+                        winnersn: winner && winner.split(" ").length > 1 ? winner.match(/\b(\w)/g).join('') : winner,
+                        winnerln: winner,
+                    };
+                },
+                templates: {
+                    item: "base_expanding_item",
+                    elClass: {
+                        tileExpand: "zci-cricket-footer"
+                    },
+                    options: {
+                        content: Spice.cricket.content,
+                        moreAt: false
```

Comment:
A link to Yahoo's Cricket score page. We should be sending them to the source for more info and in this case Yahoo has a place where they're using this data. https://cricket.yahoo.com/